src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -182,9 +182,6 @@ def get_security_updates(self):
         code, out = self.env_layer.run_command_output(self.prep_security_sources_list_cmd, False, False)
         if code != 0:
             self.composite_logger.log_warning(" - SLP:: Return code: " + str(code) + ", Output: \n|\t" + "\n|\t".join(out.splitlines()))
-            error_msg = 'Unexpected return code (' + str(code) + ') from command: ' + self.prep_security_sources_list_cmd
-            self.status_handler.add_error_to_status(error_msg, Constants.PatchOperationErrorCodes.PACKAGE_MANAGER_FAILURE)
-            raise Exception(error_msg, "[{0}]".format(Constants.ERROR_ADDED_TO_STATUS))
```

Comment:
Raising this comment to bring attention to these changes.

These lines were added as a quick fix for the ubuntu issue: https://github.com/Azure/LinuxPatchExtension/pull/162
Should be okay to remove now. 

And, for the other ask of issuing a warning, we do log this as warning currently. (have no other way I can think of, for marking this as a warning)